src/main/java/org/prebid/server/auction/BidResponseCreator.java

```diff
@@ -445,6 +446,14 @@ private Future<List<BidderResponse>> invokeProcessedBidderResponseHooks(List<Bid
                 .map(CompositeFuture::list);
     }
 
+    private Future<List<BidderResponse>> invodeAllProcessedBidResponsesHook(List<BidderResponse> bidderResponses,
```

Comment:
Invoke 🤗

Reply:
Yeah, nice catch. Fixed locally, not pushed yet :D